src/client/components/AddPeopleToRoom/AddPeopleToRoom.stories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+import AddPeopleToRoom from '../../containers/AddPeopleToRoom/AddPeopleToRoom';
```

Comment:
We don't usually put containers in storybook - it is intended to showcase components.